code/modules/clothing/head/armor.dm

```diff
@@ -465,6 +465,12 @@
 	icon_state = "maska"
 	armor_down = list(melee = 55, bullet = 55, energy = 0, bomb = 45, bio = 0, rad = 0) // best what you can get, unless you face lasers
 
+/obj/item/clothing/head/armor/faceshield/altyn/maska/tripoloski
+	name = "striped maska helmet"
+	desc = "Someone has painted a Maska in the Gopnik style"
+	icon_state = "altyn_tripoloski"
+	style = STYLE_HIGH
```

Comment:
Armor with style? Cringe!

Reply:
It is all the rage with the youth of eris. Those are premium Adidas stripes! Really I added it to differentiate it from the normal maska, but if this is the first armor to have style it can be removed to not start an armor style revolution.

Reply:
> Armor with style? Cringe!

Yes, triad jacket is cringe, I agree.